src/gluonts/dataset/pandas.py

```diff
@@ -182,6 +182,9 @@ def __iter__(self) -> Iterator[DataEntry]:
     def __len__(self) -> int:
         return len(self._data_entries)
 
+    def __str__(self) -> str:
+        return f"PandasDataset with {len(self)} series, freq={self.freq}"
```

Comment:
I would prefer something like
```suggestion
        return f"PandasDataset<size={len(self)}, freq={self.freq}>"
```